common/src/main/java/org/wildfly/swarm/ts/common/docker/Docker.java

```diff
@@ -21,6 +21,7 @@
     private final String image;
     private final List<String> ports = new ArrayList<>();
     private final Map<String, String> environmentVariables = new HashMap<>();
+    private final Map<String, String> variables = new HashMap<>();
```

Comment:
So, these are not variables, these are command arguments. `var` doesn't say anything.